kernel/Makefile

```diff
@@ -143,12 +137,6 @@ push: push_$(2)$(3)$(4)
 forcepush: forcepush_$(2)$(3)$(4)
 show-tags: show-tag_$(2)$(3)$(4)
 
-# 'docker build' with the FROM image supplied as --build-arg
```

Comment:
I wouldn't delete this comment yet, since it sounds like by removing content trust we can simplify the bits below

Reply:
Addressed. Added a FIXME to the top to show that the comment isn't current anymore.